src/Client.cpp

```diff
@@ -120,7 +120,15 @@ void TClient::SetCarData(int Ident, const std::string& Data) {
     DeleteCar(Ident);
 }
 
+// TODO: count unicycles and vehicles different #246    sla-ppy
+// PROPOSAL: separate unicycles completely from vehicles???
 int TClient::GetCarCount() const {
+    std::unique_lock lock(mVehicleDataMutex);
+    for (auto& v : mVehicleData) {
```

Comment:
Please put a short(ish) comment here explaining the *reasoning* for this solution: For example, it should outline what this fixes, and why this is the fix that was chosen ("it works lol").
You can then remove the comment on L129.